plugins/file-uploads/upload-handler.js

```diff
@@ -176,9 +176,13 @@ UploadTask.prototype.changeCountUnchanged = function(title) {
 // - the tiddler has not changed since we uploaded it
 UploadTask.prototype.makeCanonicalURITiddler = function(title) {
 	var tiddler = this.wiki.getTiddler(title),
-		canonical_uri = this.tiddlerInfo[title].canonical_uri;
+		canonical_uri = this.tiddlerInfo[title].canonical_uri,
+		fields = this.tiddlerInfo[title].fields;
+	
+	var options = Object.assign({},{text:"",_canonical_uri:canonical_uri},fields);
```

Comment:
@FSpark You can skip assigning the `options` variable here and instead in line 185 just pass as follows to `new  $tw.Tiddler`:

`this.wiki.addTiddler(new $tw.Tiddler(tiddler,{text:"",_canonical_uri:canonical_uri},fields));`